apps/inviwo/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 #--------------------------------------------------------------------
 # Inviwo Qt Application
-project(inviwo)
+project(Inviwo)
```

Comment:
do we have to do uppercase? we generally have all targets as lowercase, and I would like to keep it like that.

Reply:
Testing to see if the search path is case-dependent. The target name will also be the executable name

Reply:
There are cmake settings to control that.

Reply:
Ok, just read the add_executable documentationhttps://cmake.org/cmake/help/latest/command/add_executable.html
But that is a better option in that case

Reply:
See now that one can use OUTPUT_NAME